### PR TITLE
Book pagination

### PR DESCRIPTION
**Pull request by Nikname2303**

### CODE REVIEW

**Review by fedaktaras — APPROVED** [review]
Looks good to me

File: src/main/java/com/example/bookshop/controller/BookController.java [hunk 1]
```diff
@@ -17,33 +20,39 @@
...
         return bookService.save(requestDto);
     }
 
+    @Operation(summary = "Get book by id")
     @GetMapping("/{id}")
     public BookDto getBookById(@PathVariable Long id) {
         return bookService.findById(id);
     }
 
+    @Operation(summary = "Delete book by id")
```
Comment (andrii-hoienko) [inline src/main/java/com/example/bookshop/controller/BookController.java]: Please add description  to be consistent

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Nice, left minor comments

**Review by karma-o — APPROVED** [review]
:+1:

**Pull request merged**